packages/visx-tooltip/src/tooltips/TooltipWithBounds.tsx

```diff
@@ -20,19 +33,26 @@ function TooltipWithBounds({
   top: initialTop = 0,
   unstyled = false,
   nodeRef,
+  portalContainerPosition,
+  visualParentRect: visualParentBounds = parentBounds,
```

Comment:
rather than introducing a new prop (this component is getting super complex), could `TooltipInPortal` simply overwrite `parentRect` and it's always assumed to be the visual parent?

Reply:
I did try to overwrite the `parentRect` prop when I worked on this, but whatever I passed from here was being overwritten by the `withBoundingRects` decorator, which is what sets the `parentRect` prop. The only way I see around this would be to refactor the `TooltipWithBounds`/`withBoundingRects` couple, but that feels risky and beyond the scope of this PR. 😕